src/datepicker/DatePicker.ts

```diff
@@ -5,6 +5,7 @@ import { ariaAttr, callAllHandlers } from "@chakra-ui/utils";
 
 import { DATE_PICKER_KEYS } from "./__keys";
 import { DatePickerStateReturn } from "./DatePickerState";
+import { first } from "lodash";
```

Comment:
let's install lodash-es instead of lodash... it has lot of issues with treeshaking.

Reply:
This is not used! My VsCode auto imported I believe